File: w3/t2/person.js
class Person {
  /** @type {string} */
  first_name;
  /** @type {string} */
  last_name;
  /** @type {number} */
  #age;

  constructor(fname, lname, age) {
    this.first_name = fname;
    this.last_name = lname;
    this.#age = age;
  }

  /** @returns {number} */
  getAge() {
    return this.#age;
  }

  ageUp() {
    this.#age++;
  }

  /** @returns {string} */
  getFullname() {
    return `${this.first_name} ${this.last_name}`;
  }

  printFullname() {
    console.log(`${this.first_name} ${this.last_name}`);
  }
}

module.exports = Person;

File: w3/t1/person.py
class Person:
  first_name: str
  last_name: str
  __age: int

  def __init__(self, fname: str, lname: str, age: int) -> None:
    self.first_name = fname
    self.last_name = lname
    self.__age = age
  
  def getAge(self) -> int:
    return self.__age

  def ageUp(self) -> None:
    self.__age += 1
    return None

  def getFullname(self) -> str:
    return f"{self.first_name} {self.last_name}"

  def printFullname(self) -> None:
    print(f"{self.first_name} {self.last_name}")